tests/Unit/Crypt/RSA/CreateKeyTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2013 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ */
+
+use phpseclib\Crypt\RSA;
+
+class Unit_Crypt_RSA_CreateKeyTest extends PhpseclibTestCase
+{
+    protected $privatekey;
+    protected $publickey;
+
+    public function testCreateKey()
+    {
+        extract(RSA::createKey(512));
+        $this->assertInstanceOf('\phpseclib\Crypt\RSA', $privatekey);
+        $this->assertInstanceOf('\phpseclib\Crypt\RSA', $publickey);
+        $this->assertNotEmpty("$privatekey");
+        $this->assertNotEmpty("$publickey");
+
+        $this->publickey = $publickey;
+        $this->privatekey = $privatekey;
+    }
+
+    /**
+     * @depends testCreateKey
+     */
+    public function testEncryptDecrypt()
```

Comment:
This method should better receive it's dependencies as an argument instead of having them passed via properties.

```public function testEncryptDecrypt($args)
{
     list($publickey, $privatekey) = $args;
```

The function above needs to return those arguments:
`return array($publickey, $privatekey);`